lintly/builds.py

```diff
@@ -77,7 +77,7 @@ def execute(self):
         patch = self.get_pr_patch(diff)
         self._diff_violations = self.find_diff_violations(patch)
         logger.info('Lintly found diff violations in {} files'.format(len(self._diff_violations)))
-
+        
```

Comment:
Looks like you've added unrelated whitespace to this PR. When you are staging files it's always a good idea to look at each file being staged and not just run `git add all` or `git add .`.  VSCode has a good view for seeing the diffs in all the staged files.

Reply:
fixed in a force push